binstar_client/utils/projects/__init__.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import logging
+import os
+from os.path import basename, isdir, join, relpath
```

Comment:
every time you need a new function from os.path you will have to import it here. if you stop using it, you will have to remove it from here. why not `use os.path.<func>`?